src/config.rs

```diff
@@ -76,6 +77,12 @@ impl Config {
                     .long("endpoint-addr")
                     .env("ONETUN_ENDPOINT_ADDR")
                     .help("The address (IP + port) of the WireGuard endpoint (remote). Example: 1.2.3.4:51820"),
+                Arg::with_name("host-addr")
```

Comment:
I'm tempted to make the name more explicit, such as `endpoint-bind-addr`. Thoughts?

Reply:
It's more descriptive if you think about it, but can also be more confusing when we already have ``endpoint_addr``. I'll leave it up to you; if you think it's a better idea I'll change it.

Reply:
Yeah I'd prefer `endpoint-bind-addr`. Thanks!